projects/kerttu-global/src/app/+identification/identification-results/identification-results.component.ts

```diff
@@ -22,16 +23,23 @@ export class IdentificationResultsComponent implements OnInit {
   siteStats$: Observable<IIdentificationSiteStat[]>;
   userStats$: Observable<IIdentificationUserStatResult>;
   speciesStats$: Observable<IIdentificationSpeciesStat[]>;
+  ownSpeciesStats$: Observable<IIdentificationSpeciesStat[]>;
```

Comment:
Not a problem with this PR specifically, but I don't like using hungarian notation for interfaces. Also Angular, or any other lib we use doesn't do it either. Eg. this one should just be `IdentificationSpeciesStat` imo. Maybe this issue should be raised during dev talk sometime.

Reply:
Sounds good to me, it would be better if the same notation would be used everywhere